assembly/cdh5.14.2/src/main/resources/meta/splicemachine_env.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -ex
+
+
+#The following is written to aid local testing
+if [ -z $PARCELS_ROOT ] ; then
+    export MYDIR=`dirname "${BASH_SOURCE[0]}"`
+    PARCELS_ROOT=`cd $MYDIR/../.. &&  pwd`
+fi
+PARCEL_DIRNAME=${PARCEL_DIRNAME-SPLICE}
+
+MYLIBDIR=${PARCELS_ROOT}/${PARCEL_DIRNAME}/lib
+
+[ -d $MYLIBDIR ] || {
+    echo "Could not find splice parcel lib dir, exiting" >&2
+    exit 1
+}
+
+[ -d ${PARCELS_ROOT}/SPARK2 ] || {
+    echo "Could not find SPARK2 parcel lib dir, exiting" >&2
+    exit 1
+}
+
+PREPENDSTRING=`echo ${MYLIBDIR}/*.jar ${PARCELS_ROOT}/SPARK2/lib/spark2/jars/*.jar | sed 's/ /:/g'`
+echo "prepending $PREPENDSTRING to HBASE_CLASSPATH_PREFIX"
+if [ -z $HBASE_CLASSPATH_PREFIX ] ; then
+    export HBASE_CLASSPATH_PREFIX="${PREPENDSTRING}"
+else
+    export HBASE_CLASSPATH_PREFIX="${PREPENDSTRING}:${HBASE_CLASSPATH_PREFIX}"
+fi
```

Comment:
You can simplify the entire `if/else` block above to a single line (i.e., no need for the if/else at all):
```bash
export HBASE_CLASSPATH_PREFIX="${PREPENDSTRING}${HBASE_CLASSPATH_PREFIX:+:$HBASE_CLASSPATH_PREFIX}"
```
To explain, the expression `${HBASE_CLASSPATH_PREFIX:+:$HBASE_CLASSPATH_PREFIX}` will either resolve to an empty string if `HBASE_CLASSPATH_PREFIX` is either undefined or empty, or to the evaluation of everything following the `:+`, which is `:$HBASE_CLASSPATH_PREFIX`.

In other words, if `HBASE_CLASSPATH_PREFIX` is empty or undefined, the entire expression (the right side of the assignment) evaluates only to the value of `PREPENDSTRING`. Otherwise it evaluates to the value of `PREPENDSTRING`, followed by a colon (`:`), followed by the value of `HBASE_CLASSPATH_PREFIX`.

For reference, see https://www.tldp.org/LDP/abs/html/parameter-substitution.html